src/core/use-toaster.ts

```diff
@@ -34,58 +34,64 @@ export const useToaster = (toastOptions?: DefaultToastOptions) => {
     };
   }, [toasts, pausedAt]);
 
-  const handlers = useMemo(
-    () => ({
-      startPause: () => {
-        dispatch({
-          type: ActionType.START_PAUSE,
-          time: Date.now(),
-        });
-      },
-      endPause: () => {
-        if (pausedAt) {
-          dispatch({ type: ActionType.END_PAUSE, time: Date.now() });
-        }
-      },
-      updateHeight: (toastId: string, height: number) =>
-        dispatch({
-          type: ActionType.UPDATE_TOAST,
-          toast: { id: toastId, height },
-        }),
-      calculateOffset: (
-        toast: Toast,
-        opts?: {
-          reverseOrder?: boolean;
-          gutter?: number;
-          defaultPosition?: ToastPosition;
-        }
-      ) => {
-        const { reverseOrder = false, gutter = 8, defaultPosition } =
-          opts || {};
+  const startPause = useCallback(() => {
```

Comment:
I memoized all of these handlers individually because I needed `updateHeight` to be properly memoized instead of changing every time one of the other handlers change.